src/AutoRest.CSharp/Mgmt/AutoRest/MgmtOutputLibrary.cs

```diff
@@ -68,6 +68,10 @@ public MgmtOutputLibrary(CodeModel codeModel, BuildContext<MgmtOutputLibrary> co
 
         public IEnumerable<TypeProvider> Models => SchemaMap.Values;
 
+        public ResourceGroupExtensions ResourceGroupExtensions => new ResourceGroupExtensions(_context);
```

Comment:
remove this and resourcegroupextensionresource, they won't be needed after we add the operation group to the resource constructor

Reply:
Done